projects/gnomad/src/client/DownloadsPage.js

```diff
@@ -150,8 +150,10 @@ export default () => (
       .
     </p>
     <p>
-      <strong>Update (March 6, 2019): gnomAD 2.1.1 released.</strong> The links below have been
-      updated. Note that the coverage files have not changed for this release.{' '}
+      <strong>Update, March 6, 2019: gnomAD 2.1.1 released</strong>{' '}
+      <ExternalLink href="https://00e9e64bacde0f4f049f3635851097635a4f883459fddb76b5-apidata.googleusercontent.com/download/storage/v1/b/gnomad-public/o/release%2F2.1.1%2FREADME.txt?qk=AD5uMEsfPZE1aw1s-LhrZ2E-zhsceAPN3N_DhQnL2gOOSkCcFe2X2wrExiu_yQkMenKchIpuxFBkBM53VOgS0BFvipQOC8Dm6Ztu0a-Zx-LtdHmr1KCxUnqfuev3QAilQR4O-23wV55R5b0aQ9Ft7FlgAqx-iZrAKejgJ6AgYRCjuBKRbIeYqFNSM8cO6404dHJ8_hPM6JpeI1pgjnNczuhw-Dr1X8djZrGF65znYstFkAkxsY2dQIl0xARTeR-OOWx9iWP7Xcb2p62vIeX2T2aZ6PkYzVLlcJ2YFXpozpXxjuZ15OQPXBmRsRlpFlBPb1rS0ZQ5LZvsZgFtuo5cNNpSCzxphHuZfCQ3QNGbB2ipuSOqAQ6m98BAEKmi4jbgeSw_WSy_UxDIIK6SA15xnaQrwICOm-n18AL7xp6_2TAOcywVha7h_APnazThRZgRIR_3_5cBV17-OletXSExfI8WUkdtENlju6EhaO86tj0FXE7XGcJtH4orhVJeqRzjc6HB9zFPuq6MbD_xT_s-QuTEL1kV0T2EeQz2nvqwwCAcweP4El0Rbk_EFryIyi35AyQarLrv41Q4oQHfetAOProHaRl8o0_jnBY35KPGY_FVUz1V773lErJGYmysa8UUBNOD1PEVzg_-ujKBES6ppkx9rUK-MVfqC7HA7iLw08PEmI9V4jVt_iv1Llenwl3GC8u8W-_tZNtUa5WKaTJQDHFCzfm6SxLwF5r4-yOCLJcxyPhRJsfTP98h2LVM4jreG7CkxLJEiRs1eUGS9J3CT8aRfdawPn8BUA">
```

Comment:
Where is this URL from?  The public link for this file in the storage browser is "https://storage.googleapis.com/gnomad-public/release/2.1.1/README.txt"

Reply:
Sorry, that URL was generated when I accessed the file through the cloud console. Fixed now.